src/src/components/list-menu/ListMenuItem.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <li :id="id">
+      <fa :icon="icon" class="menu-icon" ></fa>
```

Comment:
Codacy has a fix for the issue: ['class' should be on a new line.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
      <fa :icon="icon"
```